javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,40 @@
+const width = 3;
+const height = 65;
+const area = width * height;
+console.log('Value of area:', area);
+console.log('Type of area:', typeof area);
+
+const bill = 14;
+const payment = 12;
+const change = bill - payment;
```

Comment:
Your `payment` should be larger than your `bill`. You'll also need to swap line 9. `payment - bill`